### ISSUE TITLE
else: is not automatically dedented

### ISSUE THREAD

**Issue by jontaa** [kbrose/vsc-python-indent#19]
When doing an if-else statement in python the if and the else should be on the same indention level.

This is how it looks today when doing no manual indentation
```
if value == True:
    print("something")
    else: 
        print("something")
```

This is how the expected behavior should be
```
if value == True:
    print("something")
else: 
    print("something")
```

Right now I need to manually indent the else: one step to the left.

**Comment by kbrose** [comment]
@jontaa I think some formatting may have been lost. You can reference https://guides.github.com/features/mastering-markdown/ to learn more about using markdown on GitHub.

**Comment by jontaa** [comment]
Sorry for that. I have updated the formatting now.

**Comment by kbrose** [comment]
This will likely be resolved by #27, but the change is fairly intrusive in a part of the code I don't know how to unit test so I want to do more testing before I release.

**Comment by kbrose** [comment]
This feature was released in v0.9.0
